src/HealthDataAIServices/HealthDataAIServices.Autorest/README.md

```diff
@@ -62,11 +62,28 @@ directive:
   - where:
       variant: ^CreateViaIdentity.*$
     remove: true
+  # Remove cmdlet for PrivateLink resource
+  - where:
+      subject: .*PrivateLink.*
+    remove: true
+  # Try to replace any occurrence of Ai with AI
+  - where:
```

Comment:
This aims to change your cmdlets. But your cmdlets would not contain these but `Deid`

Reply:
Removed